gym_super_mario_bros/smb_env.py

```diff
@@ -206,29 +211,39 @@ def _player_state(self):
         return self.ram[0x000e]
 
     @property
-    def _is_dying(self):
+    def _kills(self) -> int:
+        """Return the number of killed enemies."""
+        current_enemy_status = [self.ram[address] for address in _ENEMY_STATUS_ADDRESSES]
+        for i in range(len(_ENEMY_STATUS_ADDRESSES)):
+            if self._prev_enemy_status[i] == 0 and current_enemy_status[i] in _ENEMY_KILLED_STATUS:
+                self._kill_count += 1
+        self._prev_enemy_status = current_enemy_status
+        return self._kill_count
+
```

Comment:
When status is 0, mario do not kill.
If status in `_ENEMY_KILLED_STATUS`, the enemy is killed by some reason. (stomped, fireball, star, or pushing Koopa Troopa)